app/uk/gov/hmrc/nisp/views/nispMain.scala.html

```diff
@@ -1,57 +0,0 @@
-@*
-* Copyright 2016 HM Revenue & Customs
-*
-* Licensed under the Apache License, Version 2.0 (the "License");
-* you may not use this file except in compliance with the License.
-* You may obtain a copy of the License at
-*
-*     http://www.apache.org/licenses/LICENSE-2.0
-*
-* Unless required by applicable law or agreed to in writing, software
-* distributed under the License is distributed on an "AS IS" BASIS,
-* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-* See the License for the specific language governing permissions and
-* limitations under the License.
-*@
-
-@import uk.gov.hmrc.play.frontend.auth.AuthContext
-@import uk.gov.hmrc.nisp.controllers.auth.NispUser
-
-@(browserTitle: String = Messages("nisp.title"),
-    loginDetailsHeader: Option[Html] = None,
-    pageTitle: Option[String] = None,
-    userLoggedIn : Boolean = false,
-    sidebarLinks: Option[Html] = None,
-    sidebarClasses: Option[String] = None,
-    h1Classes:Option[String]=None,
-    showTitleHeaderNav: Boolean = true,
-    showBetaBanner: Boolean = true,
-    pageScripts: Option[Html] = None,
-    contentHeaderExtra: Option[Html] = None,
-    articleClasses: Option[String] = None,
-    analyticsAdditionalJs: Option[Html] = None,
-    articleEnabled: Boolean = true)(mainContent: Html)(implicit request:Request[_], authContext: Option[AuthContext] = None, user: Option[NispUser] = None, partialRetriever: uk.gov.hmrc.play.partials.CachedStaticHtmlPartialRetriever)
-
-@contentHeader = {
-    @loginDetailsHeader
-    @contentHeaderExtra
-}
-
-@main(
-    title = browserTitle,
-    sidebarLinks = sidebarLinks,
-    sidebarClasses = sidebarClasses,
-    contentHeader = Some(contentHeader),
-    userLoggedIn = userLoggedIn,
-    showTitleHeaderNav = showTitleHeaderNav,
-    showBetaBanner = showBetaBanner,
-    pageScripts = pageScripts,
-    articleClasses = articleClasses,
-    analyticsAdditionalJs = analyticsAdditionalJs,
-    articleEnabled = articleEnabled) {
-        @if(pageTitle.isDefined) {
-        <h1 class="heading-large top-title-heading"> @pageTitle </h1>
```

Comment:
Why have we removed this functionality?
It seems a bit silly to have ```<h1 class="heading-large top-title-heading"> @pageTitle </h1>```in every template.